src/main/java/com/plzgraduate/myongjigraduatebe/takenlecture/application/service/delete/DeleteTakenLectureServiceById.java

```diff
@@ -0,0 +1,29 @@
+package com.plzgraduate.myongjigraduatebe.takenlecture.application.service.delete;
+
+import org.springframework.transaction.annotation.Transactional;
+
+import com.plzgraduate.myongjigraduatebe.completedcredit.application.usecase.GenerateOrModifyCompletedCreditUseCase;
+import com.plzgraduate.myongjigraduatebe.core.meta.UseCase;
+import com.plzgraduate.myongjigraduatebe.takenlecture.application.port.DeleteTakenLecturePort;
+import com.plzgraduate.myongjigraduatebe.takenlecture.application.usecase.delete.DeleteTakenLectureUseCase;
+import com.plzgraduate.myongjigraduatebe.user.application.usecase.find.FindUserUseCase;
+import com.plzgraduate.myongjigraduatebe.user.domain.model.User;
+
+import lombok.RequiredArgsConstructor;
+
+@UseCase
+@Transactional
+@RequiredArgsConstructor
+public class DeleteTakenLectureServiceById implements DeleteTakenLectureUseCase {
```

Comment:
```suggestion
public class DeleteTakenLectureService implements DeleteTakenLectureUseCase {
```

Reply:
[3881bfc](https://github.com/Myongji-Graduate/MyongjiGraduate-BE/pull/249/commits/3881bfcf79e561c0f454ab6d84f255a8b340e731)